integration-tests/api/helpers/cart-seeder.js

```diff
@@ -720,6 +721,54 @@ module.exports = async (connection, data = {}) => {
 
   await manager.save(cartWithItemPercDiscount)
 
+  const salesChannel = manager.create(SalesChannel, {
```

Comment:
suggestion: In the new tests that you create, could we use the factory approach? That way we can also keep the existing working as it is